openduck-py/openduck_py/routers/voice.py

```diff
@@ -279,7 +279,7 @@ async def connect_daily(
         session_id=session_id,
         record=record,
         input_audio_format="int16",
-        tts_config=TTSConfig(provider="elevenlabs", voice_id=voice_id),
+        tts_config=TTSConfig(provider="local", voice_id=voice_id),
```

Comment:
can we revert the changes in this file before merging? in prod it's nice to have recording and 11 labs voice for now

Reply:
ya